docs/reference/providers/backend.md

```diff
@@ -0,0 +1,76 @@
+# K8sGPT AI Backends
+
+A Backend or a Provider is a service that provides access to the AI language model. There are many different backends available for K8sGPT. Each backend has its own strengths and weaknesses, so it is important to choose the one that is right for your needs.
+
+Currently we have a total of 4 backends available:
+- OpenAI
```

Comment:
Could you please add the links to the main "website" for each backend?

Reply:
Sure, will do.